DemoQA_test_02/Class1.cs

```diff
@@ -0,0 +1,55 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.DevTools.V117.BackgroundService;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DemoQA_test_02
+{
+    public class Class1
+    {
+        //2-ame teste įvesti neteisingo formato email ir patikrinti, kad email laukelis tapo apvestas raudonai.
+
+        [Test]
+
+        public void Test02()
+        {
+            IWebDriver driver = new ChromeDriver();
+            driver.Url = "https://demoqa.com/text-box";
+
+            string expectedResultTop = "rgba(255, 0, 0, 1)";
+            string expectedResultLeft = "rgba(255, 0, 0, 1)";
+            string expectedResultRight = "rgba(255, 0, 0, 1)";
+            string expectedResultBottom = "rgba(255, 0, 0, 1)";
```

Comment:
`expectedColorBorderTop` ir t.t. būtų geresni pavadinimai. Be to, į 'border' galima žiūrėti kaip į vientisą atributą, nebūtina skaidyti jį į keturias dalis.